server/routes/reviewsTable.ts

```diff
@@ -106,18 +106,18 @@ function parseSorting(sortString: string | undefined, orderString: string | unde
   // default to sorting by next review date if not provided or invalid
   if (!sortOptions.includes(sortString as Sort)) {
     // eslint-disable-next-line no-param-reassign
-    sortString = 'nextReviewDate'
+    sortString = 'NEXT_REVIEW_DATE'
   }
 
   // default ordering if not provided or invalid
   if (!orderOptions.includes(orderString as Order)) {
     // these columns prefer descending order:
-    if (['positiveBehaviours', 'negativeBehaviours', 'acctStatus'].includes(sortString)) {
+    if (['POSITIVE_BEHAVIOURS', 'NEGATIVE_BEHAVIOURS', 'ACCT_STATUS'].includes(sortString)) {
       // eslint-disable-next-line no-param-reassign
-      orderString = 'descending'
+      orderString = 'DESC'
     } else {
       // eslint-disable-next-line no-param-reassign
-      orderString = 'ascending'
+      orderString = 'ASC'
```

Comment:
I'm guessing the API doesn't care about the case of the `sort` query param?

Reply:
actually, turns out it's extremely fussy about case. that's why i've renamed all these params. i would prefer lower camel case, but that makes the incentives-api more awkward to build and wouldn't follow Spring Data convention